showcase/pom.xml

```diff
@@ -0,0 +1,192 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.cloud</groupId>
+  <artifactId>gapic-showcase-parent</artifactId>
+  <packaging>pom</packaging>
+  <version>0.0.1-SNAPSHOT</version>
+  <name>GAPIC Showcase Client Core Parent</name>
+  <description>
+    Java idiomatic client for Google Cloud Platform services.
+  </description>
+
+  <parent>
```

Comment:
Now that we have a proper multi module set up, you may want to consider using the parent bom like other [modules](https://github.com/googleapis/gapic-generator-java/blob/83766f1c180b756e58bf8c147ea697fbe51a4d13/gax-java/pom.xml#L12) did, so that we don't have to list all the versions. It could be improved in a separate PR though, @suztomo may also have some ideas.